### PR TITLE
Make the version updates and the install tool MySQLi compatible

### PR DESCRIPTION
**Pull request by leofeyer**
If you want to reproduce the initial problem, make sure to set `'DATABASE_URL' => 'mysqli://…'` in your .env file.

### CODE REVIEW

**Comment by ausi** [comment]
Should we duplicate https://github.com/contao/contao/blob/5e17a4acbc0370e8c956f2d3f8db94d4e9648cbb/.github/workflows/ci.yml#L264-L267 to use mysqli too?

```yaml
 - name: Run the functional tests with mysqli
   run: vendor/bin/phpunit --testsuite=functional --colors=always 
   env: 
       DATABASE_URL: mysqli://root:root@127.0.0.1:3306/contao_test 
```
_(all three occurrences)_

File: core-bundle/src/Migration/Version408/Version480Update.php [hunk 1]
```diff
@@ -348,26 +334,21 @@ public function runImportantPart(): void
...
-                        importantPartY = :y,
-                        importantPartWidth = :width,
-                        importantPartHeight = :height
-                    WHERE
-                        id = :id
-                ')
-                ->executeStatement($updateData)
-            ;
+            $this->connection->executeStatement('
+                UPDATE
```
Comment (ausi) [inline core-bundle/src/Migration/Version408/Version480Update.php]:
```suggestion
            $this->connection->executeStatement(
                'UPDATE
```
Seems to be inconsistent? _(just a nitpick, feel free to ignore)_

File: installation-bundle/src/InstallTool.php [hunk 2]
```diff
@@ -384,14 +384,35 @@ public function persistAdminUser(string $username, string $name, string $email,
             '=' => '&#61;',
         ];
 
-        $statement->executeStatement([
```
Comment (ausi) [inline installation-bundle/src/InstallTool.php:387]: `$statement = $this->connection->prepare("…` on line 355 needs to be removed I think.

Comment (leofeyer) [reply installation-bundle/src/InstallTool.php:387]: Fixed in e7ef92549dcec88463628f5bd274df6c13879fa9.

File: core-bundle/src/Migration/Version408/Version480Update.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Migration/Version408/Version480Update.php]: Fixed in 1068a12064e06624a77d7660b92bfa3ec5bec387.

**Review by ausi — DISMISSED** [review]

**Review by ausi — DISMISSED** [review]

**Review by ausi — APPROVED** [review]

**Comment by leofeyer** [comment]
Thanks a lot for your help @ausi!

**Pull request merged**